modules/installation-user-infra-machines-advanced.adoc

```diff
@@ -244,6 +244,62 @@ This `APPEND` option preserves partition 6:
 coreos.inst.save_partindex=6
 ----
 
+[id="installation-user-infra-machines-advanced_storage_luks_{context}"]
+== Using encrypted LUKS storage
+Here is an example of how to configure a LUKS device at `/var/lib/data`:
```

Comment:
It looks as though these examples were copied from [FCOS docs](https://docs.fedoraproject.org/en-US/fedora-coreos/storage/#_encrypted_storage_luks), but they don't make sense here.  These are RCCs rather than Ignition configs or MachineConfigs (you can tell by the existence of `variant`/`version`), but this section doesn't explain how to use FCCT, and in general we don't want FCCT widely used in 4.7.  Also the `rhcos/4.7` variant/version pair is invalid.

Stepping back a bit, what's the goal of adding this section?

Reply:
I'm going to pull this section. 

Late in the day, I was looking at including examples from upstream and found this: https://docs.fedoraproject.org/en-US/fedora-coreos/storage/ and noticed that LUKS encryption followed separate /var partitioning in the upstream doc repo, so that is why I was starting from this spot.

However, this PR is a WIP, so apologies for the confusion. I'll take another look tomorrow and restart based on your feedback.